src/dummywallet.cpp

```diff
@@ -24,7 +24,7 @@ class DummyWalletInit : public WalletInitInterface {
 
     // Dash Specific WalletInitInterface InitCoinJoinSettings
     void AutoLockMasternodeCollaterals() const override {}
-    void InitCoinJoinSettings() const override {}
+    void InitCoinJoinSettings(CJClientManager& clientman) const override {}
```

Comment:
This `CJClientManager& clientman` (and at all other instances of InitCoinJoinSettings) can be made const

Reply:
Good catch! Resolved in latest push.